Indicators/PercentagePriceOscillator.cs

```diff
@@ -54,7 +54,7 @@ protected override decimal ComputeNextValue(IndicatorDataPoint input)
         {
             var value = base.ComputeNextValue(input);
 
-            return (IsReady && Slow != 0) ? 100 * value / Slow : 0m;
+            return Slow != 0 ? 100 * value / Slow : 0m;
```

Comment:
Do you think 100 is the infinity case with the PPO? Maybe when slow is zero it should return 100?